deployments/fpga_admissionwebhook/crd/bases/fpga.intel.com_acceleratorfunctions.yaml

```diff
@@ -19,18 +19,13 @@ spec:
   scope: Namespaced
   validation:
     openAPIV3Schema:
-      description: AcceleratorFunction is a specification for an Accelerator Function
-        resource provided by a FPGA-based programmable hardware accelerator.
+      description: AcceleratorFunction is a specification for an Accelerator Function resource provided by a FPGA-based programmable hardware accelerator.
```

Comment:
What's your output for `controller-gen --version`?
With v0.4.1 all the long lines are broken to fit my screen.

Reply:
```
# controller-gen --version
Version: v0.4.1
```

Reply:
Weird

Reply:
Yes, I was curious of this myself. If we find the reason behind it, I could redo the commits.

Reply:
> Yes, I was curious of this myself. If we find the reason behind it, I could redo the commits.

I get correctly split lines:
```
+    schema:
+      openAPIV3Schema:
+        description: AcceleratorFunction is a specification for an Accelerator Function
+          resource provided by a FPGA-based programmable hardware accelerator.
```

Interesting indeed.

Reply:
@ozhuraki I could reproduce it only with their master branch. Can you try to rebuild controller-gen this way and try again:
```
git clone https://github.com/kubernetes-sigs/controller-tools.git
git checkout controller-tools/cmd/controller-gen
git reset --hard v0.4.1
go install
```

Reply:
fast path is:
```GO111MODULE=on go get -u sigs.k8s.io/controller-tools/cmd/controller-gen@v0.4.1```

Reply:
@mythi tried this, got an error:
```
$ GO111MODULE=on go get -u sigs.k8s.io/controller-tools/cmd/controller-gen@v0.4.1
go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.4.1: can't request explicit version of path in main module
```

It looks like it depends on where it runs. after changing directory to $HOME it worked.

Reply:
@bart0sh 

Thanks! However, this is what I get:

```
$ git clone https://github.com/kubernetes-sigs/controller-tools.git
$ cd controller-tools
$ git checkout v0.4.1
$ go build ./cmd/controller-gen
$ ./controller-gen --version
Version: (devel)
```

@mythi 

> ```$ GO111MODULE=on go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.4.1```

I tried with ```go get -u ...```, this actually did it, i.e. controller-gen splits the lines on make generate.

I will recreate commits. Thanks to you both!

Reply:
> It looks like it depends on where it runs. after changing directory to $HOME it worked.

@bart0sh strange, it works for me from elsewhere too. The binary is installed to `$GOPATH/bin`.